charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -130,9 +130,7 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `master.deploymentAnnotations`           | Annotations for master deployment           | `{}`                                      |
 | `master.lifecycle`                | Lifecycle specification for master-container | Not set                           |
 | `master.priorityClassName`        | The name of a `priorityClass` to apply to the master pod | Not set               |
-| `master.admin.existingSecret`     | The name of an existing secret containing the admin credentials. | `""`|
-| `master.admin.userKey`            | The key in the existing admin secret containing the username. | `jenkins-admin-user` |
```

Comment:
Which benefit does it bring to remove those keys?

Reply:
@torstenwalter `master.admin.existingSecret` is not removed.
only the `master.admin.userKey` is removed because now the chart enforces that the existing secret should have keys ADMIN_PASSWORD & ADMIN_USER

Reply:
I've seen that and asked myself which benefit it brings to force usage of those keys?

Reply:
@torstenwalter jenkins container required env `ADMIN_PASSWORD` and `ADMIN_USER `, right?
previously, we were able to pass these envs from existing secret using `env` in container yaml.
now, while using `envFrom` instead of `env`, we cannot use keys in different names and hence chart expects the keys in existing secret should have these names

Reply:
Then I rephrase my question which benefit do we have by using `envFrom` instead of `env`.

Don't get e wrong I am a big fan of simplifying the chart. I just want to understand why we are doing things.

Breaking changes would be a good candidate to target the jenkins-3-0-0 branch.

Reply:
@torstenwalter Its good to have clear and same understanding. am happy to explain. no worries
This PR is a fix to this issue #94 , where having credentials in env in deployment yaml exposes it . 
Hence I thought of not revealing admin credentials as plain text in deployment yaml for security purposes.

Reply:
it doesn't fix the issue as far as i'm aware, it will still be exposed on the system info page.

this just removes it from the pod.yaml

Reply:
thanks @timja . I will close this PR. 
so the fix should be credentials should not be mounted as env variable in container, right? Is that possible here in jenkins?

Reply:
We could either not configure admin user at all and let setup wizard run or configure user via CasC and make use of https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/secrets.adoc

Reply:
Secrets can be mounted as files. By doing this, you provide credentials to the container, you don't expose them to the environment and you can benefit from filesystem-based access control (ie. by not allowing some users to read this file at all, based on unix permissions)

Reply:
You are right with this. Do you want to create a PR which implements it?